packages/nimble-components/CONTRIBUTING.md

```diff
@@ -77,15 +77,34 @@ Before building a new component, 3 specification documents need to be created:
 
 ## Develop new components
 
-### Marking a component as in development
+### Marking a component as experimental / in-development / incubating
 
-If a component will require multiple pull requests before having a complete and stable API, it should be marked as "in-development" to indicate to clients that they shouldn't start using it yet. To do this:
+If a component is not ready for general use, it should be marked as "experimental" to indicate that status to clients. A component could be in this state if it:
+
+-   is still in development
+-   is currently application-specific and hasn't yet been generalized for broader use
+-   is missing important features like interaction design, visual design, accessibility, or framework integration
+
+Experimental contributions may compromise on the above capabilities but they still must abide by other repository requirements like:
+
+-   coding conventions (element naming, linting, code quality)
+-   unit and Chromatic test coverage
+-   Storybook documentation
+-   semantic versioning
+-   TODO: NAMING
+
+To do this:
 
 1. In the component status table, set its status to ⚠️
-2. In the component Storybook documentation, add a red text banner to the page indicating that the component should not be used
-3. Consider placing the component implementation in a sub-folder named `experimental` so that it will be obvious when importing it that it is incomplete
+2. In the component Storybook documentation:
+    - add a red text banner to the page indicating that the component is not ready for general use
+    - start the Storybook name with "Experimental/" so that it appears in a separate section of the documentation page
```

Comment:
Should we try and align on one term? I kinda like incubating (feels like just getting started / ambiguous enough that you should just ask) over in development (feels like it's part of expected development activity) or experimental (feels like just trying things out).

Reply:
Kinda aligns with the apache incubator: https://en.wikipedia.org/wiki/Apache_Incubator

>The Apache Incubator project serves on the one hand as a temporary container project until the incubating project is accepted and becomes a top-level project of the Apache Software Foundation or becomes subproject of a proper project such as the Jakarta Project or Apache XML.

Reply:
Coupled with Molly's comment, stuff under incubating does not use major semver, only minor or patch

Reply:
Updated to use the incubating terminology and remove the semver guidance (documented elsewhere).